test/Npgsql.Tests/Types/CompositeTests.cs

```diff
@@ -453,8 +453,8 @@ public async Task Table_as_composite([Values] bool enabled)
             Assert.ThrowsAsync<NotSupportedException>(DoAssertion);
             // Start a transaction specifically for multiplexing (to bind a connector to the connection)
             await using var tx = await connection.BeginTransactionAsync();
-            Assert.Null(connection.Connector!.DatabaseInfo.CompositeTypes.SingleOrDefault(c => c.Name.Contains(table)));
-            Assert.Null(connection.Connector!.DatabaseInfo.ArrayTypes.SingleOrDefault(c => c.Name.Contains(table)));
+            Assert.That(connection.Connector!.DatabaseInfo.CompositeTypes.SingleOrDefault(c => c.Name.Contains(table)), Is.Null);
```

Comment:
I definitely like that this removes AreEqual, replacing them with `Is.EqualTo` instead (making it clear what's the expected and what's the actual), but for Assert.Null it seems to mainly make the assertion more verbose... Is this something we have to do (because the analyzers recommend it), or can we not to do it?

Reply:
Same feeling here.
NUnit v4 enforces the fluent syntax even if sometimes it makes the assertions more verbose.
The previous assert methods are still available in a `ClassicAssert` static class, but I would recommend to not mix both solutions in the same project.